os/drivers/seclink/Make.defs

```diff
@@ -21,6 +21,7 @@ ifeq ($(CONFIG_SECURITY_LINK_DRV),y)
 
 CSRCS += seclink_drv.c
 CSRCS += seclink_drv_key.c seclink_drv_auth.c seclink_drv_common.c seclink_drv_ss.c seclink_drv_crypto.c
+CSRCS += seclink_drv_utils.c
```

Comment:
I think you should add `select TIMER` in framework/src/seclink/Kconfig for timer usage.
If user enable seclink only, build errors always will occur.